software/lazydogpile.livecodescript

```diff
@@ -0,0 +1,954 @@
+      --      put the name of button i of group "mygroup" after field 1
+      set the opaque  of button i of card 1 of stack "Answer Dialog" to false
+   end repeat
+   
+   put empty into sMain
+   put empty into sNowDoc
+   put empty into sNowItem
+   put empty into sPointerTags
+   -- initiate development logging
+   put "initiate" into xX["write"];    devLog xX
+   quickModeSetup sMain,sNowDoc,sNowItem,sPointerTags -- quickiator
+   put empty into sMainShort
+   adjList sMain,sMainShort
+   put empty into sGraphData
+   tempConvertData sMainShort,sGraphData,sNowDoc -- backend alpha and frontend alpha used different data structures
+   put empty into sGraphDrawing
+   --   put 8 into sNowFocus 
+   --   -- find a good node to focus on; highest is probably the last place they worked
+   --   put the keys of sMainShort[sNowDoc] into tKeys
+   --   sort tKeys numeric descending
+   --   repeat for each line tAkey in tKeys
+   --      if sMainShort[sNowDoc][tAkey]["type"] is "state" then
+   --         put tAkey into sNowFocus
+   --         exit repeat
+   --      end if
+   --   end repeat
+   --   -- now middleOut can start
+   middleOut sGraphData,sGraphDrawing,sNowFocus -- displayerang.livecode
+   put the width of stack "howstrBeta" into tSizes["width"]
+   put the height of stack "howstrBeta" into tSizes["height"]
+   put 40 into sLayerSize
+   put 60 into sNodeSize
+   put 8 into sLinkSize
+   set the width of field "preState" stack "howstrBeta" to sNodeSize
+   set the height of field "preState" stack "howstrBeta" to sNodeSize
+   set the width of field "preChange" stack "howstrBeta" to sNodeSize
+   set the height of field "preChange" stack "howstrBeta" to sNodeSize
+   put sLayerSize into tSizes["layer"]
+   put sNodeSize into tSizes["node"]
+   put sLinkSize into tSizes["link"]
+   displayManager sGraphDrawing,sGUIstuff,tSizes,sNowFocus,sMouseHistory -- displayerang.livecode
+   put "unsaved" into sMain[sNowDoc][0]["saveStatus"]
+   --   put false into sBusy
+   set the cBusy of stack "HowstrBeta" to false
+   put empty into field "filenameLabel" stack "HowstrBeta"
+   -- initial geometry for node detail window
+   put 400 into sNodeDetailsGeometry["startWidth"]
+   put 400 into sNodeDetailsGeometry["startHeight"]
+   put "5,30,200,395" into sNodeDetailsGeometry["tagDataList"]
+   put "205,30,395,51" into sNodeDetailsGeometry["tagLabel"]
+   put "205,56,395,395" into sNodeDetailsGeometry["editData"]
+   put "219,41" into sNodeDetailsGeometry["loadImgButt"]
+   put "256,41" into sNodeDetailsGeometry["clearImgButt"]
+   put "300,41" into sNodeDetailsGeometry["expandImgButt"]
+   put "340,41" into sNodeDetailsGeometry["setCaptureButt"]
+   put "372,41" into sNodeDetailsGeometry["snapCaptureButt"]
+   -- initial geometry for screen capture area window
+   put 400,400,800,800 into sCaptureAreaGeometry["captureAreaRect"]
+end setupAsOf3FEB2016
+
+on resizeNodeDetails pWidth,pHeight
```

Comment:
@matthewmaier This handler should be moved to the script of stack NodeDetails or to a behavior script.